st2api/st2api/controllers/v1/keyvalue.py

```diff
@@ -97,7 +97,17 @@ def get_one(self, name, requester_user, scope=None, user=None, decrypt=False):
         key_ref = get_key_reference(scope=scope, name=name, user=user)
         extra = {"scope": scope, "name": name, "user": user, "key_ref": key_ref}
         LOG.debug("GET /v1/keys/%s", name, extra=extra)
-
+        LOG.audit(
+            "User %s decrypted the value %s ",
+            user,
+            name,
+            extra={
+                "user": user,
+                "scope": scope,
+                "key_name": name,
+                "operation": "decrypt",
+            },
+        )
```

Comment:
I think we also need this audit log in `get_all()` (this one is in `get_one()`.

Reply:
added this as well !